quickstarts/kubernetes/nginx/config.yml

```
@@ -0,0 +1,51 @@
+name: NGINX Ingress Controller
```

This just needs to called `slug` and be hyphenated lowercase
```suggestion
slug: nginx-ingress-controller
```

---

Oops, I'll get that fixed!